shadow.html

```diff
@@ -54,6 +54,12 @@
                   0 5px 5px -3px rgba(0, 0, 0, 0.4);
     };
 
+    --shadow-elevation-12dp: {
```

Comment:
A combination of interpolation and "what looks right," assuming 8dp and 16dp are "correct." :/